docs/source/developer/how_to_extend.rst

```diff
@@ -16,10 +16,10 @@ just replicate and adapt that code to get a basic derivative of
 *openwisp-radius* working.
 
 If you want to add new users fields, please follow the `tutorial to extend the
-openwisp-users <https://github.com/openwisp/openwisp-users/tree/master#extend-openwisp-users>`_.
+openwisp-users <https://github.com/openwisp/openwisp-users/blob/master#extend-openwisp-users>`_.
```

Comment:
`tree` -> `blob` was an unnecessary redirect performed by github, all `tree` -> `blob` are to avoid this redirect.